testingLocations/README.md

```diff
@@ -0,0 +1,17 @@
+# To Runs
+
+Run the script in the root folder
```

Comment:
When you say root folder do you mean in RDE-covid-resource-guide? Because it looks like you created a react project in /testingLocations.

The root folder typically indicates the highest folder in the project. I believe in this instance it would be the repo folder RDE-covid-resource-guide.